frontend/src/stores/auth.js

```diff
@@ -3,8 +3,36 @@ import { defineStore } from 'pinia'
 
 import ApiService from '@/common/apiService'
 import AuthService from '@/common/authService'
+import FacilityService from '@/services/facilityService'
 import { useAppStore } from '@/stores/app'
-import { APPLICATION_INTAKE_TYPES } from '@/utils/constants'
+import { APPLICATION_INTAKE_TYPES, OFM_PROGRAM_CODES } from '@/utils/constants'
+
+/*
+  A facility can apply for a core application if it satisfies these conditions:
+  1. There is an Open Intake or the facility is in the allowed facility list of a Limited Intake
+  2. If the facility is in the CCOF program, their program start date must >= 1 year from the current date.
+*/
+function intakeWindowCheckForAddApplication(facility) {
```

Comment:
This is being picky but can we put these utility functions at the bottom of the file? It was confusing at first as to why they were in the store until I realized they were used in getUserInfo().

Reply:
it's updated. thanks for your suggestions!